src/main/java/uk/gov/hmcts/reform/sscs/ccd/service/CcdService.java

```diff
@@ -87,10 +87,18 @@ public SscsCaseDetails createCase(SscsCaseData caseData, String eventType, Strin
         return createCcdCaseService.createCase(caseData, eventType, summary, description, idamTokens);
     }
 
+    /**
+     * @deprecated  Use UpdateCcdCaseService.UpdateCaseV2 methods as this could cause potential data issues.
```

Comment:
I think we should also refer to trigger event v2 where only event triggering is required